src/components/Card.js

```diff
@@ -1,14 +1,55 @@
 import styled from 'styled-components';
 import ScreenReaderOnly from './ScreenReaderOnly';
 import StyledButton from './StyledButton';
+import useToggle from '../hooks/useToggle';
 
 export default function Card({ question, answer, onTrashClick, _id }) {
+  const [solution, toggleSolution] = useToggle(false);
   return (
     <FileCard>
-      <h2>Frage: </h2>
-      <span>{question}</span>
-      <h2>Antwort: </h2>
-      <span>{answer}</span>
+      <ScreenReaderOnly>
+        <h2>Frage: </h2>
+      </ScreenReaderOnly>
```

Comment:
do you want to use the h2 as label for the styled question? What is the utility of it? Maybe you could connect it to the question with an "ariaLabelledBy"

Reply:
> do you want to use the h2 as label for the styled question? What is the utility of it? Maybe you could connect it to the question with an "ariaLabelledBy"

I wasn't sure if it's common to label a span, I think like this it's read in normal textflow as if it was displayed. I want to test it in the evening :)